### DISCUSSION TITLE
Configuration backup

### DISCUSSION THREAD

**Discussion by zwannie** [2Abendsegler/GClh#1961] [Questions] [answered]
How can i backup the configuration?

**Comment by 2Abendsegler** [comment, marked as answer]
Hi zwannie,
you can download the configuration or save it in DropBox. Press the F10 key on your keyboard and look. Maybe that's enough information?
Frank

**Reply by zwannie** [reply]
Thank you so  much, this is easier than I thouht :-)

**Comment by 2Abendsegler** [comment]
... press the F10 key if you are on a geocaching webpage.
